[framework] new phing target for changing environment (#1951)
shopsys/shopsys: docs/introduction/console-commands-for-application-management-phing-targets.md

For more information about translations, see [the separate article](../introduction/translations.md).

### Production

### environment-change

> Change environment easily by this target. 

Reviewer: ```suggestion
!!! important
    Do not use this target on your production server, because it could cause some security vulnerabilities.
```

> It can be handy in production environment, when you need to debug some functionality on testing server.

Reviewer: ```suggestion
Change environment to production, development or test using this target.
This target is hand when you need to debug some functionality on your development / testing server.
```

> Switching to development environment will also install dev dependencies of composer.

Reviewer: ```suggestion
Switching to development environment will also install composer dev dependencies.
```

> Currently, you can switch to production, development or test enviroment.

Reviewer: ```suggestion
```
